app/views/changelog.php

```diff
@@ -1,3 +1,83 @@
+<?=$release_title('3.7')?>
+<h3>End user visible changes</h3>
+<ul>
+    <li>
+        <?=$relnotes('better')?>
+        The <a href="/?sourcelocale=en-US&amp;locale=fr&amp;repo=aurora&amp;t2t=t2t&amp;recherche=Bookmarks">glossary</a> view looks better and is visually more consistent with the rest of the site (flod)
+    </li>
+    <li>
+        <?=$relnotes('better')?>
+        The application no longer needs JavaScript to work (flod)
```

Comment:
I'd rephrase this: "The application is more usable when JavaScript is disabled". Some features still aren't available, but at least navigation and most parts work.